frontend/src/e2e-playwright/utils/application-forms.ts

```diff
@@ -0,0 +1,578 @@
+import { ApplicationDetails } from 'lib-common/api-types/application/ApplicationDetails'
+import { ApplicationFormData } from 'lib-common/api-types/application/ApplicationFormData'
+import { clubFixture, daycareFixture } from 'e2e-test-common/dev-api/fixtures'
+
+function assertEquals<T>(expected: T, actual: T) {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  if (actual !== expected) throw Error(`expected ${expected}, got ${actual}`)
+}
+
+function assertBlank(actual: string | undefined) {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  if (actual !== '') throw Error(`expected empty string, got ${actual}`)
+}
+
+function assertEmpty<T>(actual: T[]) {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  if (actual.length !== 0) throw Error(`expected empty array, got ${actual}`)
+}
+
+function assertTrue(actual: boolean | undefined) {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  if (actual !== true) throw Error(`expected true, got ${actual}`)
+}
+
+function assertFalse(actual: boolean | undefined) {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  if (actual !== false) throw Error(`expected false, got ${actual}`)
+}
+
+function assertNull<T>(actual: T | undefined) {
```

Comment:
Did you mean `T | null` here? Or just `T`?

Reply:
I just copied the file from the testcafe side (and forgot to delete it), so I didn't even look at that. `T` should do just fine here.